internal/server/server.go

```diff
@@ -590,21 +601,23 @@ func newJSONDecoder(r io.Reader) *json.Decoder {
 	return decoder
 }
 
-func authenticationMiddleware(db *database.Database, inner http.HandlerFunc) http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) {
-		apiKey, err := extractBearerToken(r)
-		if err != nil {
-			http.Error(w, "Unauthorized", http.StatusUnauthorized)
-			return
-		}
+func authenticationMiddleware(db *database.Database) func(http.Handler) http.Handler {
+	return func(inner http.Handler) http.Handler {
```

Comment:
I was confused about why we were adding back the wrapper layer here, and then noticed the `chain.New` setup for the endpoint router. I think it's fine, and the `chain` bits should make it easier to layer other wrappers in the future for endpoints if we need them.